### PR TITLE
Quality: PowerShell Summit GH-3371

### LINKED ISSUE #3371: Quality: Install-AdcsWebEnrollment
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Formatting code samples, Formatting command syntax

### Article List

- https://github.com/MicrosoftDocs/windows-powershell-docs/blob/main/docset/winserver2022-ps/adcsdeployment/Install-AdcsWebEnrollment.md

### PR DESCRIPTION
**Pull request by Snozzberries**
# PR Summary

 - Resolves #3371

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsWebEnrollment.md [hunk 1]
```diff
@@ -143,12 +159,13 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ### Microsoft.CertificateServices.Deployment.Common.WEP.WebEnrollmentResult
 
 ## NOTES
-* Ensure you run Windows PowerShell as an administrator. You can use the *Force* parameter to bypass the prompt for confirmation.
-To see parameters, run the following command: `Install-AdcsWebEnrollment -?`
+
+- Ensure you run Windows PowerShell as an administrator. You can use the **Force** parameter to
+  bypass the prompt for confirmation. To see parameters, run the following command:
+  `Install-AdcsWebEnrollment -?`
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsWebEnrollment.md:165]:
```suggestion

    `Install-AdcsWebEnrollment -?`
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Snozzberries, minor suggested change. Many thanks.

**Comment by Snozzberries** [comment]
Commit applied. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2022-ps/adcsdeployment/Install-AdcsWebEnrollment.md [hunk 2]
```diff
@@ -81,11 +90,12 @@ Accept wildcard characters: False
 ```
 
 ### -Credential
-Specifies a **PSCredential** object for the CA Web Enrollment.
-To obtain a credential object, use the **Get-Credential** cmdlet.
-For more information, type `Get-Help Get-Credential`.
-If the Web Enrollment service is configured to use Standalone CA, then an account that is a member of the local Administrators on the CA is required.
-If the Web Enrollment service is configured to use an Enterprise CA, then an account that is a member of Domain Admins is required.
+
+Specifies a `PSCredential` object for the CA Web Enrollment. To obtain a credential object, use the
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/adcsdeployment/Install-AdcsWebEnrollment.md:94]:
```suggestion
Specifies a **PSCredential** object for the CA Web Enrollment. To obtain a credential object, use the
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
One minor fix - we use bold for type, parameter and property names, code-formatting for cmdlets, variables with the `$` prefix, parameters with the `-` prefix, and specific values.

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**